res/openvpn-gui-res-nl.rc

```diff
@@ -405,7 +408,7 @@ BEGIN
     IDS_ERR_AUTH_PASSWORD2STDIN "Fout bij doorgeven wachtwoord aan stdin."
     IDS_ERR_CR2STDIN "Fout bij doorgeven CR aan stdin."
     IDS_ERR_INVALID_CHARS_IN_PSW "Het nieuwe wachtwoord bevat niet toegestane tekens. " \
-                                 "Kies een nieuw wachtwoord."
+                             "Kies een nieuw wachtwoord."
```

Comment:
First line should be `Het nieuwe wachtwoord bevat niet-toegestane tekens.` - "The new password contains disallowed characters", not "The new password does not contain allowed characters".